minio/minio
# Update gorilla/websocket version

## Description
Update gorilla/websocket to release `v1.4.2` to avoid the CWE 190 reported in 
https://github.com/sonatype-nexus-community/nancy

## Motivation and Context
Avoid reported cwe

## How to test this PR?
Run `go list -m all | docker run --rm -i sonatypecommunity/nancy:latest` and confirm 
if cwe 190 is not reported.

## Types of changes

## Checklist:
